code/modules/mob/hear_say.dm

```diff
@@ -167,7 +167,22 @@
 
 /mob/proc/hear_sleep(var/message)
 	var/heard = ""
+	var/opentag = 0
+	var/closetag = 0
 	if(prob(15))
+		while(1) //Remove the HTML formatting so people don't hear colour
```

Comment:
strip_htmp_properly eliminates anything after an open tag or before a close tag, not just things inside tags, this could cause other odd behavior with custom scripts if I'm not mistaken. However I will change it if you prefer.